Source/WebCore/page/scrolling/ScrollingStateScrollingNode.h

```diff
@@ -134,6 +137,7 @@ class ScrollingStateScrollingNode : public ScrollingStateNode {
     RetainPtr<NSScrollerImp> m_verticalScrollerImp;
     RetainPtr<NSScrollerImp> m_horizontalScrollerImp;
 #endif
+    bool m_mouseIsOverContentArea { false };
```

Comment:
Group this with other bools to reduce padding.

Reply:
Fixed.